packages/components/pie-spinner/src/spinner.scss

```diff
@@ -1 +1,88 @@
 @use '@justeattakeaway/pie-css/scss' as p;
+
+*,
```

Comment:
Box sizing should be set on the application already as part of the `pie-css` library. Was it not being inherited into the component?

If not, we probably need a ticket to look at this so we don't end up duplicating this everywhere

Reply:
it's on my radar, the box-sizing set in `pie-css` doesn't work as expected with shadow DOM.

https://dev.to/konnorrogers/revisiting-box-sizing-best-practices-3del